internal/mode/static/handler_test.go

```diff
@@ -76,6 +77,7 @@ var _ = Describe("eventHandler", func() {
 			eventRecorder:       fakeEventRecorder,
 			healthChecker:       &healthChecker{},
 			controlConfigNSName: types.NamespacedName{Namespace: namespace, Name: configName},
+			metricsCollector:    collectors.NewControllerNoopCollector(),
```

Comment:
If we want, I can test that the handler calls the collector's ObserveLastEventBatchProcessTime method in these tests, but I will have to move the controllerMetricsCollector interface definition to the collectors package to avoid a cyclical import cycle. Our generated fakes can only be used with black box tests.